faq/index.md

```diff
@@ -0,0 +1,122 @@
+
+### How much time do I need to devote to the volunteering work if I choose to volunteer for PyCon India? 
+
+There isn't any strict time requirement that needs to be fulfilled by the volunteers. We value each and every contribution, no matter how big or small it is. You can choose to volunteer on your own time and availability. But, ideally, one hour a day would be more than sufficient if you can keep up the commitment and do the work assigned to you.  
+
+### Q. Do volunteers get a certificate?
+
+**No**, Pycon India does not provide any certificate to volunteers. We all volunteer on our own volition to make the conference a great experience for us and everyone else who attends it.
+
+### Q. Do attendees get a certificate of participation?
+
+**No**, PyCon India does not provide any certificates to the attendees/participants. The conference is a platform to share your experience, express your views, learn, and meet new people.
+
+
+## Ticketing FAQ
+
+---
+
+### Q. I am a speaker/volunteer at the conference. Do I need to buy a ticket?
+
+**Yes**. PyCon India is a conference built for the community, by the community. Everyone from the speakers to volunteers, to even the organizers have to pay for their own tickets to attend the conference. Nobody makes a dime during the whole event. All proceedings from the ticket sales go towards making the conference even better for everyone.
+
+Here is a beautiful article by [Jesse Noller](https://twitter.com/jessenoller) about the same http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays.
+
+### Q. What does the Conference ticket (Oct 2 and Oct 3, 2020) include?
+
+A full conference ticket for two days includes access to all the amazing keynotes, talks, and lightning talks.
+
+You will also get access to all the open space discussions and other sessions during the conference. The conference ticket holders will also be eligible to join the Devsprints.
+
+The ticket also includes all the conference goodies and swags for the attendees. 
+
+### Q. Is there a different ticket for Workshops?
+
+**Yes**, a separate ticket would have to be purchased for each workshop. The details for the workshop tickets will be updated soon. 
+
+### Q. Is there a different ticket for the Devsprints?
+
+**No**, the conference ticket includes complete access to the dev sprints. The ticket holders do not have to pay anything extra. But, the attendees will be asked to register for the dev sprints as that would help the conference with logistics. 
+
+### Q. I have bought the workshop ticket. Am I eligible to participate in DevSprints?
+
+**No**, the workshop tickets only include access to the specific workshops. You would have to buy the conference tickets to get access to the dev sprints. 
+
+### Q. There is a change in my plan so I can’t attend the conference. Can I get a refund?

```

Comment:
We would need same for workshop as well

Reply:
how about "Conference and/or workshops"?

Reply:
```suggestion
### Q. There is a change in my plan so I can’t attend the conference and/or workshops. Can I get a refund?

```

Reply:
ping @sayanchowdhury